jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/DefaultCredentialRetrieversTest.java

```diff
@@ -91,4 +100,21 @@ public void testInitAsList_all() {
     Mockito.verify(mockCredentialRetrieverFactory)
         .dockerCredentialHelper("docker-credential-credentialHelperSuffix");
   }
+
+  @Test
+  public void testInitAsList_credentialHelperPath() throws IOException {
```

Comment:
Is it worth adding a test when the file does not exist?

Reply:
`testInitAsList_all` covers this, I think.